### PR TITLE
FACES-2683 Move common rendering utilities into Util

### PR DESCRIPTION
**Pull request by stiemannkj1**

### CODE REVIEW

File: alloy/src/main/java/com/liferay/faces/alloy/render/internal/AlloyRendererBase.java [hunk 1]
```diff
@@ -44,19 +46,19 @@ public void decodeClientBehaviors(FacesContext facesContext, UIComponent uiCompo
 	@Override
 	public void encodeBoolean(ResponseWriter responseWriter, String attributeName, Boolean attributeValue,
```
Comment (stiemannkj1) [inline alloy/src/main/java/com/liferay/faces/alloy/render/internal/AlloyRendererBase.java:47]: @ngriffin7a, I prefer the name `encodeBooleanProperty` (I think it's clearer), which is why I changed it in Util. But this is not public API and it requires changes to the generator, so I left it alone.

**Comment by stiemannkj1** [comment]
@ngriffin7a, just so you know, I still need to update generator to be compatible with these changes.

File: alloy/src/main/java/com/liferay/faces/alloy/render/internal/AlloyRendererBase.java (hunk 1 above)
Comment (ngriffin7a) [inline alloy/src/main/java/com/liferay/faces/alloy/render/internal/AlloyRendererBase.java:47]: @stiemannkj1 Feel free to rename it to encodeBooleanProperty after you have updated the generator.

**Comment by ngriffin7a** [comment]
Merged into master (3.x) and backported to 2.x and 1.x (as appropriate). Thank you.

**Pull request closed without merging**